app/src/main/res/values/styles_v2.xml

```diff
@@ -580,6 +580,7 @@
     <item name="android:textColor">@color/grey0</item>
   </style>
 
+  <!--suppress AndroidLintUnusedResources -->
   <style name="Clinic.V2.TextAppearance.PatientSummary.BloodPressure.High">
```

Comment:
Can we check again where this is really an unused style? If yes, feel free to remove it.

Reply:
Only this style is not being used, but the `Clinic.V2.TextAppearance.PatientSummary.BloodPressure` and `Clinic.V2.TextAppearance.PatientSummary.BloodPressure.Normal` is being used by blood sugar item. I left to remove all at once, but I will remove it in this PR.

Reply:
Yeah, don't wait to remove unused code. If something is no longer in use, remove it.